src/backend/distributed/utils/statistics_collection.c

```diff
@@ -360,103 +347,3 @@ citus_server_id(PG_FUNCTION_ARGS)
 
 	PG_RETURN_UUID_P((pg_uuid_t *) buf);
 }
-
-
-#ifdef WIN32
```

Comment:
Discussed this with @marcocitus, we don't support Windows so no need to refactor this. Removing is easier.